tools/api-markdown-documenter/src/api-item-transforms/TsdocNodeTransforms.ts

```diff
@@ -114,6 +116,15 @@ export function _transformTsdocNode(
 		case DocNodeKind.HtmlEndTag: {
 			return transformTsdocHtmlTag(node as DocHtmlEndTag, options);
 		}
+		case DocNodeKind.InheritDocTag: {
+			options.logger?.verbose(
```

Comment:
The text makes me think this should be a warning instead of verbose.

Reply:
Oof. Copy-pasta. I actually want that one to be an error.

Reply:
```suggestion
			options.logger?.error(
```